modules/polling/context/BallotContext.tsx

```diff
@@ -178,9 +179,19 @@ export const BallotProvider = ({ children }: PropTypes): React.ReactElement => {
       }
     });
 
+    const optionIds = pollOptions.map(option => {
+      if (!Array.isArray(option)) return option;
+      if (option.length === 1) return option[0];
+      const byteArray = new Uint8Array(32);
+      option.forEach((optionIndex, i) => {
+        byteArray[byteArray.length - i - 1] = optionIndex;
+      });
+      return fromBuffer(byteArray, {}).toString();
+    });
+
     const voteTxCreator = voteDelegateContract
-      ? () => voteDelegateContract['votePoll(uint256[],uint256[])'](pollIds, pollOptions)
-      : () => polling['vote(uint256[],uint256[])'](pollIds, pollOptions);
+      ? () => voteDelegateContract['votePoll(uint256[],uint256[])'](pollIds, optionIds)
+      : () => polling['vote(uint256[],uint256[])'](pollIds, optionIds);
```

Comment:
Comparing it to how it used to be done in dai.js, it looks like we check the length of the options array to choose which `vote` method to use. Does it matter if we always use the one with `uint265[]`?

https://github.com/makerdao/dai.js/blob/dev/packages/dai-plugin-governance/src/GovPollingService.ts#L60-L66

Reply:
Nice, yeah I tested it out with one poll vote, and it worked fine. The only benefit it seems is that it's *slightly* less gas if you're only voting on one poll, but it's only like a 5% gas price reduction so it doesn't seem worth the additional complication